tenders-api/update_v7.sql

```diff
@@ -0,0 +1,18 @@
+
+/*
+Tenders DB Update script: v6 (buyer_user_details table)
+*/
+
+-- Drop table
+
+-- DROP TABLE public.buyer_user_details;
+
+CREATE TABLE public.buyer_user_details (
+	user_id int4 NOT NULL,
```

Comment:
Is this `user_id` intended to match the Jaggaer `bravoId`? (Just wondering about the `int4` type - might be safer as a varchar?